docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,428 @@
+You can check an example of the resources to apply at the [following documentation page](./inter-cluster-network.md#definition-of-the-network-configuration-configuration-crds).
+
+**The Configuration resource should be applied in both clusters** and it **must contain the pod and external CIDR of the peer cluster**. (cluster A has the CIDR config of B and vice versa).
+
+The CIDRs are defined at installation time, you can check the values of the pod and the external CIDR configured in the local cluster by running:
+
+```{code} bash
+kubectl get networks.ipam.liqo.io -n liqo
+
+NAME            DESIRED CIDR    REMAPPED CIDR   AGE
+external-cidr   10.70.0.0/16    10.70.0.0/16    3h16m
+internal-cidr   10.80.0.0/16    10.80.0.0/16    3h16m
+pod-cidr        10.244.0.0/16   10.244.0.0/16   3h16m
+service-cidr    10.96.0.0/16    10.96.0.0/16    3h16m
+```
+
+### Creating and exchanging the Wireguard keys (both clusters)
+
+To enable authentication and encryption, Wireguard requires a key pair, one for each gateway.
+
+Those keys can be created [via the Wireguard utility tool](https://www.wireguard.com/quickstart/#key-generation) or via OpenSSL like so:
+
+```{code} bash
+openssl genpkey -algorithm X25519 -outform der -out private.der
+openssl pkey -inform der -in private.der -pubout -outform der -out public.der
+
+# Get the Wireguard private key
+echo "Private key:"; cat private.der | tail -c 32 | base64
+# Get the Wireguard public key
+echo "Public key:"; cat public.der | tail -c 32 | base64
+```
+
+At this point, **in each cluster**, we will need to create a secret containing this pair of keys:
+
+```{code} yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    liqo.io/remote-cluster-id: <REMOTE_CLUSTER_ID>
+  name: gw-keys
+  namespace: <TENANT_NAMESPACE>
+type: Opaque
+data:
+  privateKey: <WIREGUARD_PRIVATE_KEY>
+  publicKey: <WIREGUARD_PUBLIC_KEY>
+```
+
+where WIREGUARD_PRIVATE_KEY and WIREGUARD_PUBLIC_KEY are the previously generated key pairs for that specific cluster (**each cluster has its pair of keys**).
+
+Additionally, each cluster should have a PublicKey resource **with the public key of the peer cluster** (cluster A has the public key of B and vice versa):
+
+```{code} yaml
+apiVersion: networking.liqo.io/v1beta1
+kind: PublicKey
+metadata:
+  labels:
+    liqo.io/remote-cluster-id: <HERE_THE_CLUSTER_ID_OF_PEER_CLUSTER>
+    networking.liqo.io/gateway-resource: "true"
+  name: gw-publickey
+  namespace: <TENANT_NAMESPACE>
+spec:
+  publicKey: <REMOTE_WIREGUARD_PUBLIC_KEY>
+```
+
+In order to make things work, make sure that the PublicKey resource has the labels:
+
+```{code} yaml
+liqo.io/remote-cluster-id: <HERE_THE_CLUSTER_ID_OF_PEER_CLUSTER>
+networking.liqo.io/gateway-resource: "true"
+```
+
+### Configuring the server gateway (cluster provider)
+
+As mentioned before, a Wireguard tunnel connects the clusters peered with Liqo.
```

Comment:
```suggestion
By default, a Wireguard tunnel connects the clusters peered with Liqo.
```

Reply:
Try to avoid "as mentioned before" or similar. Is like saying "I know that you are not paying attention, so I repeat here what I said already before".  Which is not perceived as a good message.